Module-11/Library/Documents/IDocument.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Library.Documents;
+
+public interface IDocument
+{
```

Comment:
since every document has a DocumentNumber, it should be a part of this interface (even though it is not used)